<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="sumeetsingh01" name="com.maruti.subscribe.VehicleCategories" tests="4" failures="2" timestamp="29 Apr 2020 10:21:12 GMT" time="9.980" errors="0">
  <testcase name="vehicleCategoriesTest" time="3.064" classname="com.maruti.subscribe.VehicleCategories">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Mismatch : Database and API values do not match.] but found [false],
	Mismatch : Database and API values do not match. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Mismatch : Database and API values do not match.] but found [false],
Mismatch : Database and API values do not match. expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at com.maruti.controller.VehicleCategories.VehicleCategories_Manager.validateResponse(VehicleCategories_Manager.java:144)
at com.maruti.controller.VehicleCategories.VehicleCategories_Manager.validateTestCase(VehicleCategories_Manager.java:27)
at com.maruti.subscribe.VehicleCategories.vehicleCategoriesTest(VehicleCategories.java:19)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- vehicleCategoriesTest -->
  <testcase name="vehicleCategoriesTest" time="2.427" classname="com.maruti.subscribe.VehicleCategories"/>
  <testcase name="vehicleCategoriesTest" time="3.901" classname="com.maruti.subscribe.VehicleCategories">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Mismatch : Database and Api values do not match. expected [true] but found [false],
	Mismatch : Database and Api values do not match. expected [true] but found [false],
	Mismatch : Database and Api values do not match. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Mismatch : Database and Api values do not match. expected [true] but found [false],
Mismatch : Database and Api values do not match. expected [true] but found [false],
Mismatch : Database and Api values do not match. expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at com.maruti.controller.VehicleCategories.VehicleCategories_Manager.validateResponse(VehicleCategories_Manager.java:144)
at com.maruti.controller.VehicleCategories.VehicleCategories_Manager.validateTestCase(VehicleCategories_Manager.java:27)
at com.maruti.subscribe.VehicleCategories.vehicleCategoriesTest(VehicleCategories.java:19)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- vehicleCategoriesTest -->
  <testcase name="vehicleCategoriesTest" time="0.588" classname="com.maruti.subscribe.VehicleCategories"/>
</testsuite> <!-- com.maruti.subscribe.VehicleCategories -->
